### ISSUE TITLE
Does not work under Chrome for Android (4.4.4)?

### ISSUE THREAD

**Issue by fritzmg** [ianli/raphael-sketchpad#20]
Has this been tested under Chrome for Android (Android 4.4.4)? On my Nexus 5 I am unable to draw anything within the SketchPad. It just scrolls the whole site instead.

It works fine on the iPhone though for example.

**Comment by gaucho1978** [comment]
same problem here on chrome for android 6.0
on safary for iphone 9.3.4 works fine

**Comment by gaucho1978** [comment]
the problem was solved but the version was not released.
here the changes to manually make: https://github.com/taktran/raphael-sketchpad/commit/2e91f94967159ed67e64babd42deb6cf415a5bab

**Comment by gaucho1978** [comment]
BUT I found another problem: On chrome for android, if you zoom on the webpage (pinch outside of the schetchpad) and then you draw, the drawing appears on another screen location. 
 the problem does not appear on safari for ios devices.

**Comment by rmsandu** [comment]
@gaucho1978  did you check out #6 ?

**Comment by gaucho1978** [comment]
@raluca-san , yes it works on android, but if i zoom the webpage with a pinch, and then i draw, the line is not drawn under my finger, but in another location.

Do you have a working example on internet that i can check?

**Comment by gaucho1978** [comment]
this last problem that i reported is solved with #10

**Comment by gaucho1978** [comment]
some problems solved: https://github.com/gaucho1978/raphael-sketchpad/commit/fe1c43704473ddf571e0ec668a62c655f950bb9d

it is a bug related to offset() call that appears on chrome for android after a pinch. 
I solved it.
I also allowed the ability to pinch and pan with 2 fingers and to draw with one finger.
i also made some other minor changes. see them all on the link.
